.github/ISSUE_TEMPLATE/DOCUMENTATION-REQUEST.yml

```
@@ -0,0 +1,33 @@
+name: "✏️📘 Rest-Api Documentation Request"
```

```suggestion
name: "✏️📘 Rest-API Documentation Request"
```
Check your capitalization throughout.